src/robusta/integrations/prometheus/trigger.py

```diff
@@ -87,22 +106,23 @@ def __find_node_by_ip(cls, ip) -> Optional[Node]:
                 return node
         return None
 
-    @classmethod
-    def __load_node(cls, alert: PrometheusAlert, node_name: str) -> Optional[Node]:
+    @staticmethod
```

Comment:
Any reason for changing this from a classmethod to staticmethod? It doesn't actually matter much, but you're calling another method on the same class (`__find_node_by_ip`). By keeping it as a class method and using `cls` the code will work even if someone changes the name of the class in the future.

Reply:
Same question applies to other methods below.

Reply:
I'll change it back